manifests-maistra/maistra.package.yaml

```diff
@@ -1,5 +1,5 @@
 packageName: maistraoperator
 channels:
 - name: "2.0"
```

Comment:
It's strange that we call the channel `2.0`. Shouldn't it be called `2.x`? @rcernich ?